test/e2e/specs/specs-playwright/wp-likes-spec.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+import {
+	BrowserHelper,
+	DataHelper,
```

Comment:
Why do we import those 2 above in the PascalCase? I think we should reserve that for class names and constructors to avoid potential confusion.

Reply:
I do not recall in which PR, but in one of the very early PRs the convention was set.

[This StackOverflow](https://stackoverflow.com/questions/29601960/typescript-use-pascalcasing-or-camelcasing-for-module-names) post is a bit old but it appears the convention is to use PascalCase for modules and classes?

Reply:
Thanks for the context, makes sense to me! I guess the most important thing is to stay consistent 👍